app/controllers/api/orders_controller.rb

```diff
@@ -19,5 +24,12 @@ def serialized_orders(orders)
         each_serializer: Api::Admin::OrderSerializer
       )
     end
+
+    def order
+      @order ||= Spree::Order.
+        where(number: params[:id]).
+        includes(line_items: { variant: [:product, :stock_items, :default_price] }).
```

Comment:
Nice.I just did some work on a variant query, and eager-loading `:product, :stock_items, :default_price` made a massive difference. We should take a look and maybe add this to the other orders pages at some point.